### PR TITLE
clarify README w.r.t. installation and configuration

### PR DESCRIPTION
**Pull request by slhck**
Addresses #4 and provides detailed instructions on setting up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,10 +65,11 @@ The following options can be passed to the plugin via `docker plugin set` (all n
 - **`apikey`** (**required**): authentication token to use when accessing the Hetzner Cloud API
 - **`size`** (optional): size of the volume in GB (default: `10`)
 - **`fstype`** (optional): filesystem type to be created on new volumes. Currently supported values are `ext{2,3,4}` and `xfs` (default: `ext4`)
-- **`prefix`** (optional): prefix to use when naming created volumes (default: `docker`)
+- **`prefix`** (optional): prefix to use when naming created volumes; it will be separated from the created volume name with a hyphen (default: `docker`)
```
Comment (costela) [inline README.md]: maybe we could be even clearer with an example?
```suggestion
- **`prefix`** (optional): prefix to use when naming created volumes; the final name on the HC side will be of the form `prefix-name`, where `name` is the volume name assigned by `docker` (default: `docker`)
```

File: README.md [hunk 2]
```diff
@@ -30,20 +30,33 @@ During installation, you will be prompted to accept the plugins's privilege requ
...
-installation above):
+First, create an API key from the Hetzner Cloud console and save it temporarily.
+
+Install the plugin as described above. Then, set the API key in the plugin options, where `<apikey>` is the key you just created:
+
+```
+$ docker plugin set hetzner apikey=<apikey>
+```
+
+The plugin is then ready to be used, e.g. in a Docker Compose file, by setting the `driver` option on the docker `volume` definition (assuming the alias `hetzner` passed during installation above).
```
Comment (costela) [inline README.md]:
```suggestion
The plugin is then ready to be used, e.g. in a `docker-compose` file, by setting the `driver` option on the docker `volume` definition (assuming the alias `hetzner` passed during installation above).
```

File: README.md [hunk 3]
```diff
@@ -30,20 +30,33 @@ During installation, you will be prompted to accept the plugins's privilege requ
...
+
+For example, when using the following Docker Compose volume definition in a project called `foo`:
```
Comment (costela) [inline README.md]:
```suggestion
For example, when using the following `docker-compose` volume definition in a project called `foo`:
```

**Review by costela — COMMENTED** [review]
Thanks for the PR!

Just added some minor comments, PTAL.

File: README.md (hunk 1 above)
Comment (slhck) [reply README.md]: Good suggestion! :+1:

**Comment by slhck** [comment]
I tried to resolve everything, again using GitHub GUI, please squash-merge if you accept the PR. Thanks!

**Comment by costela** [comment]
Great! Thanks again! :+1:

**Pull request merged**